### PR TITLE
qa/workunit: print the detail commands excuted in the scripts

### PR DESCRIPTION
**Pull request by lxbsz**
This will make it to be much easier when debugging the qa tests failures.

---

When debugging the qa test failures at [1], it's hard to exactly get which command was it running when it failed.

[1]: https://pulpito.ceph.com/yuriw-2023-04-05_14:51:06-fs-wip-yuri5-testing-2023-04-04-0814-distro-default-smithi/7232784/

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/workunits/fs/full/subvolume_clone.sh [hunk 1]
```diff
@@ -19,7 +19,6 @@ set -ex
 # Note that the failed clones would be in retry loop and it's state would be 'pending' or 'in-progress'.
 # It's state is not updated to 'failed' as the config update to gets ENOSPACE too.
 
-set -e
```
Comment (dparmar18) [inline qa/workunits/fs/full/subvolume_clone.sh:22]: We don't want to set to `-ex` in `subvolume_clone.sh`?

Comment (lxbsz) [reply qa/workunits/fs/full/subvolume_clone.sh:22]: There is a `set -ex` in line#2 of this file already.

Comment (dparmar18) [reply qa/workunits/fs/full/subvolume_clone.sh:22]: ah, okay. i saw every other place you added `ex` removing `e`, so thought it should be here too.

**Review by dparmar18 — APPROVED** [review]
LGTM

**Comment by vshankar** [comment]
@lxbsz would it make sense to do this for _all_ workunits?

**Comment by lxbsz** [comment]
> @lxbsz would it make sense to do this for _all_ workunits?

Almost all these will be used by cephfs tests. If this is not okay I will only revise this for `qa/workunits/kernel_untar_build.sh`.

**Comment by vshankar** [comment]
> > @lxbsz would it make sense to do this for _all_ workunits?
> 
> Almost all these will be used by cephfs tests. If this is not okay I will only revise this for `qa/workunits/kernel_untar_build.sh`.

I'm fine with adding command details. Was just curious :)

**Comment by lxbsz** [comment]
jenkins retest this please

**Comment by dparmar18** [comment]
jenkins test api

**Comment by vshankar** [comment]
jenkins retest this please

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
jenkins retest this please

**Pull request merged**